public/app/core/components/Login/UserSignup.tsx

```diff
@@ -1,8 +1,12 @@
 import React, { FC } from 'react';
 import { LinkButton, VerticalGroup } from '@grafana/ui';
 import { css } from 'emotion';
+import { getConfig } from 'app/core/config';
 
 export const UserSignup: FC<{}> = () => {
+  let href = getConfig().verifyEmailEnabled ? '/verify' : '/signup';
+  href = getConfig().appSubUrl + href;
+
```

Comment:
Nitpick: avoid using `let` when you can use `const` instead.
```suggestion
  const href = getConfig().verifyEmailEnabled ? `${getConfig().appSubUrl}/verify` : `${getConfig().appSubUrl}/signup`;
```